CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ This can also be enabled programmatically with `warnings.simplefilter('default',
 ### Fixed
 * [`fpdf.drawing.DeviceCMYK`](https://py-pdf.github.io/fpdf2/fpdf/drawing.html#fpdf.drawing.DeviceCMYK) objects can now be passed to [`FPDF.set_draw_color()`](https://py-pdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.set_draw_color), [`FPDF.set_fill_color()`](https://py-pdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.set_fill_color) and [`FPDF.set_text_color()`](https://py-pdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.set_text_color) without raising a `ValueError`: [documentation](https://py-pdf.github.io/fpdf2/Text.html#text-formatting).
 * individual `/Resources` directories are now properly created for each document page. This change ensures better compliance with the PDF specification but results in a slight increase in the size of PDF documents. You can still use the old behavior by setting `FPDF().single_resources_object = True`.
+* Wrapping words on spaces now considers zero-width spaces (U+200B) in addition to regular spaces (' '), addressing issues with word-wrapping for languages like Thai (as per [#1190](https://github.com/py-pdf/fpdf2/issues/1190))
```

Comment:
Maybe also mention something like "all common space symbols" here instead of NBSP specifically.

Reply:
Hi @gmischler I've made changes as per your suggests above to Changelog, and using a single list for spaces.  Hope this is useful!